From: Richard Purdie <richard.purdie@linuxfoundation.org>
Date: Tue, 26 Mar 2013 13:56:22 +0000
Subject: [PATCH] opkg-build: Remove bashism in sh script

Otherwise this fills logs with output on systems with dash as /bin/sh,
we might as well execute cleanly.

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
---
 opkg-build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opkg-build b/opkg-build
index f8c6151..90a7111 100755
--- a/opkg-build
+++ b/opkg-build
@@ -239,7 +239,7 @@ rm $tmp_dir/tarX
 
 echo "2.0" > $tmp_dir/debian-binary
 
-if (( $opkext == 1)); then 
+if [ $opkext -eq 1 ]; then 
 	pkg_file=$dest_dir/${pkg}_${version}_${arch}.opk
 else
 	pkg_file=$dest_dir/${pkg}_${version}_${arch}.ipk
